src/terrain/generation/biome.cpp

```diff
@@ -234,7 +233,7 @@ Biome::get_json_data(const std::string& biome_name) {
     if (materials_file.has_value())
         materials_file.value() >> materials_data;
 
-    return {biome_name, biome_data, materials_data};
+    return {biome_name, std::move(biome_data), std::move(materials_data)};
```

Comment:
Take `biome_name` as a `std::string` (no reference), move it into the struct, and then move when calling `get_json_data`